src/Differ/MapDiffer.php

```diff
@@ -176,6 +178,8 @@ private function isAssociative( array $array ) {
 	 * spot differences between array values. Very weird the native
 	 * function just ignores these...
 	 *
+	 * @SuppressWarnings(PHPMD.ShortVariable)
```

Comment:
`$id` is an absolutely valid variable name. A "configured minimum length" of 3 is not helpful. Please don't let us start adding such weird suppress tags all over our code base.